.github/workflows/ci.yml

```
@@ -0,0 +1,21 @@
+name: CI
+on: pull_request
+jobs:
+  test:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: Brightspace/third-party-actions@actions/checkout
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+        with:
+            node-version-file: .nvmrc
+      - name: Install dependencies
+        run: npm install

```

`npm ci` is more appropriate for CI jobs
```suggestion
        run: npm ci
```

---

agree, was just testing stuff. will eventually change it to ci